manifests/role/archivematica.pp

```diff
@@ -13,6 +13,12 @@
   include nebula::profile::duo
   include nebula::profile::networking
 
+  # TSM to be included via puppet starting with Debian 9 onwards
+  if $facts['os']['family'] == 'Debian' \
```

Comment:
Archivematica systems are all Ubuntu. Also, there are no older puppet-managed instances, so all with this role can have the TSM profile.